spec/components/sections/view_spec.rb

```diff
@@ -17,7 +17,7 @@ module Sections
     context "apply draft trainee in review" do
       let(:section) { :course_details }
       let(:status) { :review }
-      let(:trainee) { create(:trainee, :in_progress, :with_apply_application) }
+      let(:trainee) { create(:trainee, :in_progress, :with_apply_application, start_academic_cycle: AcademicCycle.all.first) }
```

Comment:
I don't think you need the `all` here unless there is some performance reason?

```suggestion
      let(:trainee) { create(:trainee, :in_progress, :with_apply_application, start_academic_cycle: AcademicCycle.first) }
```